src/OrchardCore.Modules/OrchardCore.Tenants/OrchardCore.Tenants.csproj

```diff
@@ -9,6 +9,7 @@
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Environment.Navigation\OrchardCore.Environment.Navigation.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Admin.Abstractions\OrchardCore.Admin.Abstractions.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.DisplayManagement\OrchardCore.DisplayManagement.csproj" />
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.Data.Abstractions\OrchardCore.Data.Abstractions.csproj" />
```

Comment:
Why?

Reply:
Because one view reference it and was failing to compile. This knowing that views are precompiled when building the related module and in the context of this module.

Reply:
Do the current themes just work as it? Don't they require more references?

Reply:
I will return on this branch and try them.

But i think it is ok because when there is a missing reference the theme fails to build